### DISCUSSION TITLE
Fine-tuning JSON format

### DISCUSSION THREAD

**Discussion by ignaciot** [urchade/GLiNER#200] [General]
Hi there, thanks for providing such a straightforward finetuning notebook. I'm experimenting with fine-tuning GLiNER and I was wondering, is the `data.json` file in the finetuning example provided in any specific standard format? (and is there a tool that readily outputs that specific JSON format?)

Asking before writing a custom CONLL/whatever converter to that expected JSON structure...

**Comment by davidress-ILW** [comment]
@ignaciot I would recommend using the GLiNER fine-tuning branch instead of the fine-tuning jupyter notebook.  

https://github.com/urchade/GLiNER/tree/training

This branch was easy to follow given the steps listed. Note that it uses GLiNER 0.2.2, which is both behind and ahead of the main branch, so I recommend setting up a private environment. Look at the pilener.json file for the format the training data needs to be in.

**Reply by ignaciot** [reply]
Excellent, thanks a lot!
